backend/app/api/v1/controllers/helpRequests/remove.ts

```diff
@@ -1,30 +1,29 @@
 import { Request, Response, NextFunction } from "express";
 
-import * as helpRequestsServices from "../../../../services/helpRequests"
+import * as helpRequestsServices from "../../../../services/helpRequests";
 
 export const remove = async (req: Request, res: Response, next: NextFunction) => {
     try {
-        const {help_request_id} = req.params
-        await helpRequestsServices.remove(help_request_id)
-        res.status(204).send()
+        const { help_request_id } = req.params;
+        await helpRequestsServices.remove(help_request_id);
```

Comment:
How could we be sure that operation was successful? 
You could checkout my implementation 

https://github.com/serhii-fsi/be-news/blob/main/src/controllers/delete-comment.js
(and also check the model: removeComment)